packages/@monocdk-experiment/assert/jest.ts

```diff
@@ -0,0 +1,93 @@
+import { Stack } from '@aws-cdk/core';
```

Comment:
How does this import and similar imports in this package not break? I don't see what command will do the rewriting...?

Reply:
Added rewriting